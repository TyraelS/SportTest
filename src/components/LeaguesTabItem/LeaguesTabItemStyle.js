import styled from 'styled-components';

const LeaguesTabItemStyle = styled.div`
  display: flex;
  justify-content: space-between;
  align-items: center;
  > * {
    padding: 10px;
  }
`;

export default LeaguesTabItemStyle;
