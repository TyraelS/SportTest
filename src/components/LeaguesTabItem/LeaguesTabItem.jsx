import React from 'react';

import LeaguesTabItemStyle from './LeaguesTabItemStyle';

const displayName = 'LeaguesTabItem';

export default function LeaguesTabItem({ name, country }) {
  return (
    <LeaguesTabItemStyle>
      <div>{name}</div>
      <div>{country}</div>
    </LeaguesTabItemStyle>
  );
}

LeaguesTabItem.displayName = displayName;
