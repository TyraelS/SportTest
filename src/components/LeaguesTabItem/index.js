import LeaguesTabItem from './LeaguesTabItem';

export default LeaguesTabItem;
