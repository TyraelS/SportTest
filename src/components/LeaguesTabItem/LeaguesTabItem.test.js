import React from 'react';
import { shallow } from 'enzyme';
import LeaguesTabItem from './LeaguesTabItem';

describe('Given the LeaguesTabItem component with provided props', () => {
  describe('when the component is rendered', () => {
    const component = shallow(
      <LeaguesTabItem name='Test name' country='GBR' />
    );

    it('should match the snapshot', () => {
      expect(component).toMatchSnapshot();
    });
  });
});
