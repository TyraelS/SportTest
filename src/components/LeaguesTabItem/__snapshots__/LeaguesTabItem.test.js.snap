// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given the LeaguesTabItem component when the component is rendered should match the snapshot 1`] = `
<ForwardRef(styled.div)>
  <div>
    Test name
  </div>
  <div>
    GBR
  </div>
</ForwardRef(styled.div)>
`;
